projects/noloan/firebase/src/main/java/hasadna/noloan/firebase/DbMessages.java

```diff
@@ -89,8 +78,8 @@ public void modifyMessage(SmsMessage newMessage) {
   public void addMessagesListener(MessagesListener messagesListener) {
     messagesListeners.add(messagesListener);
   }
-  public void removeMessagesListener(MessagesListener messagesListener)
-  {
+
+  public void removeMessagesListener(MessagesListener messagesListener) {
```

Comment:
IMO this can be removed as it has no usage...
Equivalent to the case where you wouldn't add `Getters/Setters` for all Class members, even if they don't require it.

But perhaps I'm wrong..